src/CoreBundle/private/library/classes/TCMSListManager/TCMSListManagerFullGroupTable.class.php

```diff
@@ -441,6 +433,37 @@ public function AddFields()
         $this->AddCustomColumns();
     }
 
+    /**
+     * @param array $fieldConfig
+     *
+     * @return null|array|string - returns the types AddColumn allows for the callback parameter
+     */
+    protected function getCellFormattingFunctionConfig(array $fieldConfig)
+    {
+        $formatFunctionName = $fieldConfig['callback_fnc'];
+
+        if ('' === $formatFunctionName) {
+            // handle UUID formatting globally if no callback method was defined
+            if ('id' === $fieldConfig['db_alias'] && 'ID' === $fieldConfig['title']) {
+                return array('TCMSRecord', 'callBackUuid');
+            }
+        }
+
+        // check if it is a standard callback, or a record object method
+        if ('gcf_' !== substr($formatFunctionName, 0, 4) && 'ccf_' !== substr($formatFunctionName, 0, 4)) {
```

Comment:
Because I don't see if the comment describes the if or the else.

Reply:
added methods (in new PR with target 6.3.x)